config.md

```diff
@@ -25,19 +25,61 @@ Changing it means creating a new derived image, instead of changing the existing
 
 ### Layer DiffID
 
-A layer DiffID is a SHA256 digest over the layer's uncompressed tar archive and serialized in the descriptor digest format, e.g., `sha256:a9561eb1b190625c9adb5a9513e72c4dedafc1cb2d4c5236c9a6957ec7dfd5a9`.
+A layer DiffID is the digest over the layer's uncompressed tar archive and serialized in the descriptor digest format, e.g., `sha256:a9561eb1b190625c9adb5a9513e72c4dedafc1cb2d4c5236c9a6957ec7dfd5a9`.
```

Comment:
> "the digest" refers to "the digest" of the layer.

A given layer can have multiple digests (see my [foo example][1]).  That means that an [indefinite article][2] is more appropriate than a [definite article][3].  Wikipedia gives the [following example][2]:

> Likewise,
>
> >    Give me the book.
>
> refers to a specific book whose identity is known or obvious to the listener; as such it has a markedly different meaning from
>
> > Give me a book.
>
> which uses an indefinite article, which does not specify what book is to be given.

In this case, the lack of an algorithm requirement means that there are several potential digests (one for each possible algorithm identifier), and we are not specifying which of those digests is intended (all are equally legal).

[1]: https://github.com/opencontainers/image-spec/pull/586#discussion_r103774864
[2]: https://en.wikipedia.org/wiki/Article_%28grammar%29#Indefinite_article
[3]: https://en.wikipedia.org/wiki/Article_%28grammar%29#Definite_article